src/rust/plugin-registry/src/server/deploy_plugin.rs

```diff
@@ -38,24 +38,15 @@ pub async fn deploy_plugin(
     let job_name = "grapl-plugin"; // Matches what's in `plugin.nomad`
     let job = {
         let job_file_hcl = static_files::PLUGIN_JOB;
-        // TODO: Deprecate this in next PR and replace with passed-in env variable
-        // ~ wimax Feb 2022
```

Comment:
did it!!!

Reply:
![](https://media.giphy.com/media/xUA7aTTTOEYbozXg2I/giphy-downsized-large.gif)